pkg/controller/atlasproject/backupcompliancepolicy.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2023 MongoDB.
```

Comment:
```suggestion
Copyright 2024 MongoDB.
```

We should use 2024 in all our new files now

Reply:
Is this something we need to add to new files from now? Maybe we will need a linter to ensure it.

Reply:
@s-urbaniak is this something worth investigating?

Reply:
The idiom (from my previous open source projects)  is to set the current year for new files only and never touch it again. it usually is effective the "creation year".

Reply:
Created CLOUDP-253160 for this